examples/data-sources/opslevel_integrations/data-source.tf

```
@@ -1,6 +1,5 @@
-data "opslevel_integrations" "all" {
-}
+data "opslevel_integrations" "all" {}
 
 output "found" {
   value = data.opslevel_integrations.all.id[0]
```

This is invalid, should be 

```suggestion
  value = data.opslevel_integrations.all.integrations[0].id
```

I think there are some other invalid plural datasource examples

---

Thanks! I'll fix this and double check the other plural datasources